torchaudio/functional.py

```diff
@@ -1193,6 +1193,52 @@ def contrast(
     return output_waveform
 
 
+def dcshift(
+        waveform: Tensor,
+        shift: float,
+        limiter_gain: float = 0.
+) -> Tensor:
+    r"""Apply a DC shift to the audio. Similar to SoX implementation.
+    This can be useful to remove a DC offset
+    (caused perhaps by a hardware problem in the recording chain) from the audio
+
+    Args:
+        waveform (Tensor): audio waveform of dimension of `(..., time)`
+        shift (float): indicates the amount to shift the audio
+            Allowed range of values for shift : -2.0 to +2.0
+        limiter_gain (float): It is used only on peaks to prevent clipping
+            It should have a value much less than 1 (e.g. 0.05 or 0.02)
+
+    Returns:
+        Tensor: Waveform of dimension of `(..., time)`
+
+    References:
+        http://sox.sourceforge.net/sox.html
+    """
+    output_waveform = waveform
+    limiter_threshold = 0.
+    use_limiter = False
+
+    if limiter_gain != 0.0:
```

Comment:
Comparing float for equality appears somewhat strange.
Instead how about setting `default_gain = None` and compare against it?

Reply:
After this change the torch script test failing with  error " Expected a default value of type float on parameter "limiter_gain" "  when using None . If I remove None and compare with float, the torch script test is working . Is there another way for making it work with None in jit  or should i revert back to float comparison?